doc/pages/communities.json

```diff
@@ -11,6 +11,12 @@
     "tags": ["women_of_colour", "black_women", "self_care", "domestic_violence", "support_groups"],
     "languages": ["en", "es"]
   },
+  {
+    "name": "if me",
+    "link": "https://www.if-me.org/",
+    "tags": ["blog", "tech_industry", "directory", "goal_tracking", "mood_tracking"],
```

Comment:
I would remove the tech_industry tag since it's a community for anyone, not just folks in the tech industry. We could replace this with an "open source" tag. A "journaling" tag would also be good to add.

Reply:
That's officer-thinking, Julia... resolved by https://github.com/ifmeorg/ifme/pull/1626/commits/0bad7c7f460672722ac31e5850afafe9cf20f0e3